samples/Directory.Build.props

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup>
+        <NeoTestVersion>3.5.17</NeoTestVersion>
```

Comment:
Shouldn't we add reference to this project?

Reply:
OK

Reply:
I find this property could be used in [test.yml](https://github.com/N3developertoolkit/neo-test/blob/60b08e4328c59798d25c133c523ecf011beef5e9/.github/workflows/test.yml#L69), changing it back to unspecified value.